gocd/bci.gocd.yaml

```diff
@@ -189,23 +191,25 @@ pipelines:
         - SLE
         environment_variables:
           OSC_CONFIG: /home/go/config/oscrc-totest-manager
+          BCI_TOKEN: '{{SECRET:[opensuse.secrets][BCI_TOKEN]}}'
```

Comment:
IBS_BCI_RELEASE_TOKEN would be a little more obvious - but let's stay with that for now.

Reply:
ok, thanks